### PR TITLE
Let Facility modules specify a custom template and add context_data

### PR DESCRIPTION
**Pull request by phycodurus**

### CODE REVIEW

**Review by jchate6 — CHANGES_REQUESTED** [review]
This has some breaking tests.

File: tom_observations/views.py [hunk 1]
```diff
@@ -198,6 +220,12 @@ def get_context_data(self, **kwargs):
 
         target = Target.objects.get(pk=self.get_target_id())
         context['target'] = target
+
+        # allow the Facility class to add data to the context
+        facility_class = self.get_facility_class()
+        facility_context = facility_class().get_facility_context_data()
```
Comment (jchate6) [inline tom_observations/views.py:226]: This line breaks the tests.

**Review by jchate6 — CHANGES_REQUESTED** [review]

File: tom_observations/views.py (hunk 1 above)
Comment (phycodurus) [reply tom_observations/views.py:226]: ack! was passling locally, but PR was missing the commit below

**Review by jchate6 — APPROVED** [review]
looks good now

**Pull request merged**